public/index.html

```diff
@@ -24,7 +54,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Lunar Hotel | Your favorite Getaway Destination</title>
```

Comment:
- [ ] **(OPTIONAL)** I can't comment in the correct line because it wasn't changed in this PR, but it would be awesome to put a personalized image as an icon to your app.

Reply:
> * [ ]  Your design is not similar to the one that was provided. In the project [requirements](https://github.com/microverseinc/curriculum-final-capstone/blob/main/projects/business_requirements.md#project-requirements), you were told to follow the layout of the provided design, and that you should change only the titles, descriptions, and photos, but your colors and typographies are different.


Hello @MuriloRoque,

The exact requirements are:
```markdown
#### Basics
- You should follow the layout of [the provided design](https://www.behance.net/gallery/26425031/Vespa-Responsive-Redesign). You should change only the titles, descriptions, and photos - in order to create a website about something other than motorcycles.
- Select a theme for your website - is it going to be a website for booking doctor appointments, booking online classes, or something else?
```

We followed the layout as a wireframe, for us to follow it exactly wouldn't be possible because of the varying themes we can choose from to customise it. Furthermore we follow the structure of the layout as best we could considering the theme we chose.


All other changes have been made.

Reply:
Hello @mmatongo, that's okay, I just thought it weren't EXACTLY like it when talking about colors and typographies, of course the theme has to be different. Nevertheless, your design is very nice and that was just my opinion, you don't need to follow it anyway 😄

Reply:
Hey, we've implemented the required changes.

Thank you for the feedback.